src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -70,43 +71,76 @@ public ClimbSubsystem() {
         motorConfig.supplyCurrLimit = MOTOR_CURRENT_LIMIT;
         motorConfig.peakOutputForward = CLIMB_SPEED;
         motorConfig.peakOutputReverse = -CLIMB_SPEED;
-
         motor.configAllSettings(motorConfig);
-
         motor.setNeutralMode(NeutralMode.Brake);
 
         setPID(P, I, D);
-
     }
 
+    /**
+     * Graciously initialize the simulation, by setting
+     * the initialization time of the motor (from 0 to full)
+     * to 1 second, and setting the maximum velocity.
+     *
+     * @param sim
+     *            The current gracious simulation
+     */
     public void simInit(PhysicsSim sim) {
         // Motor, acceleration time from 0 to full in seconds, max velocity
         sim.addTalonFX(motor, 1, SIM_FULL_VELOCITY);
     }
 
+    /**
+     * Stop the dynamic climb arm motor, graciously
+     *
+     * @param stop
+     *            Whether to stop the motor
+     */
     @Config(name = "Stop Fixed Motor")
     public void stopArm(boolean stop) {
         if (stop) {
             motor.stopMotor();
         }
     }
 
+    /**
+     * Graciously extend the dynamic climb arm,
+     * to the mid rung height
+     */
     public void extendArm() {
         setMotor(MID_RUNG_HEIGHT_INCH * ENCODER_TICKS_PER_INCH);
     }
 
+    /**
+     * Retract the dynamic climb arm, professionally,
+     * going down about 15 inches
```

Comment:
Using a specific value of 15 inches in the comment runs the risk of getting on a tread mill of keeping it up to date with the actual constant.  That is not "Good Stuff".
Who knows what it will end that up being as things continue to change.